agent/consul/state/state_store.go

```diff
@@ -155,11 +158,14 @@ func NewStateStore(gc *TombstoneGC) (*Store, error) {
 	// Create and return the state store.
 	s := &Store{
 		schema:       schema,
-		db:           db,
 		abandonCh:    make(chan struct{}),
 		kvsGraveyard: NewGraveyard(gc),
 		lockDelay:    NewDelay(),
 	}
+	s.db = &memDBWrapper{
+		db: db,
+	}
+	// TODO: s.publisher = NewEventPublisher(s, 0, 10*time.Second)
```

Comment:
We've recently gotten into the habit of tagging TODOs like this with a "slug" we can grep for later when we intend to come back and implement something later or deprecate it.

Not a huge deal in this case but it might be nice to have something like `TODO(streaming)` as a prefix so we can easily find all the TODOs that we expect to be handled by the time this basic streaming functionality is pieced back together again.